clappr/clappr-android
# Change visibility of some variables and methods

Goal
---
Change visibility of some classes and variables of Clappr

How to test
---
1 - Run Unit tests
2 - Open sample app
3 - Check if video everything is working correctly, play/pause the video, perform seek, wait for it to finish and play it again, go fullscreen and back to portrait
